<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-PSP68KB" name="com.Tour.qa.testdata.RegistrationTest" tests="34" failures="3" timestamp="2022-04-25T13:48:13 PKT" time="1153.790" errors="0">
  <testcase name="CheckFillDataInRegistrationForm" time="39.870" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="CheckInvalidEmail" time="33.806" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="CheckduplicateEmail" time="31.212" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="CheckinvalidData" time="39.582" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="emptyFormtTest" time="35.188" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="invalidPasswordTest" time="37.760" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="invalidZipCodeTest" time="35.174" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="ErrorShouldDisplyOnPrpviding10CharInMobileFildOdRegidtrationPage" time="34.954" classname="com.Tour.qa.testdata.RegistrationTest">
    <failure type="java.lang.AssertionError" message="expected [The value &#039;qwertyuiop&#039; is invalid.] but found []">
      <![CDATA[java.lang.AssertionError: expected [The value 'qwertyuiop' is invalid.] but found []
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.Tour.qa.testdata.RegistrationTest.ErrorShouldDisplyOnPrpviding10CharInMobileFildOdRegidtrationPage(RegistrationTest.java:181)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ErrorShouldDisplyOnPrpviding10CharInMobileFildOdRegidtrationPage -->
  <system-out/>
  <testcase name="invalidMobWithLeassDigit" time="37.110" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="invalidMobNumber" time="36.132" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="TestLessDigitinMob" time="36.785" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="TestPasseordWithOnlyInt" time="37.907" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="testPasswordWithOnlyString" time="37.180" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="checkFirstName" time="36.857" classname="com.Tour.qa.testdata.RegistrationTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:819)
at org.testng.Assert.assertEquals(Assert.java:829)
at com.Tour.qa.testdata.RegistrationTest.checkFirstName(RegistrationTest.java:255)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkFirstName -->
  <system-out/>
  <testcase name="checkLastName" time="40.286" classname="com.Tour.qa.testdata.RegistrationTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:819)
at org.testng.Assert.assertEquals(Assert.java:829)
at com.Tour.qa.testdata.RegistrationTest.checkLastName(RegistrationTest.java:270)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkLastName -->
  <system-out/>
  <testcase name="checkEmailnotMatch" time="41.064" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="BothPasswordFieldNotMatch" time="35.510" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="checkMandatoryFieldsOnLeavinfEmpty" time="37.429" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="VehiclesFieldWithCharacter" time="38.189" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="checkGroupSizeWithString" time="35.624" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="HomePageTitleTest" time="28.630" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="CheckBanner" time="30.083" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="validateGroupText1" time="28.693" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="validateGroupSize" time="28.342" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="validateChoosFromMenuBar" time="34.769" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="validateChoosDateFromMenuBar" time="32.585" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="HeadingValidate1" time="30.067" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="validateLoginButton" time="28.461" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="validateForgot" time="29.834" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="ValidateHeading2" time="28.517" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="ValidateHeading3" time="27.570" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="ValidateHeading4" time="27.373" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="ValidateHeading5" time="30.324" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="validateService" time="30.923" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
</testsuite> <!-- com.Tour.qa.testdata.RegistrationTest -->
